core/templates/dev/head/pages/signup/redirect_to_login_directive.html

```diff
@@ -0,0 +1,11 @@
+<div class="modal-header">
+  <h3>Redirect to Login Page</h3>
```

Comment:
Internationalize this and choose a better title that fits the purpose of the modal. Note that the modal is primarily communicating that the registration session expired; the "continue registration" is just an action that the user can take.

For the filename, use registration_session_expired_modal.html or similar (also, check the correct naming format for modal directives -- I thought they had "modal" somewhere in the name).

Reply:
Done.